### PR TITLE
Merge dev to master

### PR DESCRIPTION
**Pull request by Akhilesh2412**
This is a PR to merge dev to master branch.

Signed-off-by: Kommireddy Akhilesh<akhileshkommireddy2412@gmail.com>

### CODE REVIEW

File: answerfile.yml [hunk 1]
```diff
@@ -38,96 +38,78 @@ uplink_profiles:
...
 - display_name: TN1
   host_switches:
   - host_switch_profiles:
     - name: uplinkProfile1
       type: UplinkHostSwitchProfile
     host_switch_name: hostswitch1
-    # pnics:
-    # - device_name: "vmnic1"
-    #   uplink_name: "uplink-1"
+    host_switch_mode: STANDARD
```
Comment (madhukark) [inline answerfile.yml:48]: Dont we need host_switch_type: VDS/NVDS too? Perhaps we need to take a step back and look at all the examples in build_topology.yml + answerfile.yml. Is it being tested? If not, perhaps we should mention the last version it worked it.

File: README.md [hunk 2]
```diff
@@ -14,15 +14,27 @@ These modules are maintained by [VMware](https://www.vmware.com/).
...
 ### Supported NSX Objects/Workflows
-The modules in this repository are focused on enabling automation of installation workflows of NSX-T.
+The modules in this repository are focused on enabling automation of installation workflows of NSX-T. We have modules that support the legacy MP and new Policy API.
+
+#### MP API
+MP API modules can be used to configure an NSX resource with one-to-one mapping.
 
 ### Branch Information
 This repository has different branches with each branch providing support for upto a specific NSX-T release. Below is the list:
 * v1.1.0: NSX-T 2.4, NSX-T 2.5
```
Comment (madhukark) [inline README.md:34]: Need to add v3.0.0: NSX-T 3.0, 2.5, 2.4 to the list

File: answerfile.yml [hunk 3]
```diff
@@ -38,96 +38,78 @@ uplink_profiles:
...
-#   fabric_node_name: "Host_2"
 
+edge_transport_nodes:
+- display_name: TN1
+  host_switches:
+  - host_switch_profiles:
+    - name: uplinkProfile1
+      type: UplinkHostSwitchProfile
+    host_switch_name: hostswitch1
+    host_switch_mode: STANDARD
```
Comment (madhukark) [inline answerfile.yml:76]: same as above

File: examples/deploy_nsx_cluster/README.md [hunk 4]
```diff
@@ -16,5 +16,6 @@ to match your needs and run the playbooks in the order listed.
 
 Validated against:
 * NSX-T 2.4 GA
+* NSX-T 2.5 GA
```
Comment (madhukark) [inline examples/deploy_nsx_cluster/README.md:19]: Please add NSX-T 3.0 GA as well to this list. Or I can add it later too. Not a big deal.

File: examples/setup_infra/README.md [hunk 5]
```diff
@@ -1,24 +1,26 @@
...
-* 01_deploy_transport_zone.yml
+* 05_setup_edge_cluster.yml
+* 04_setup_transport_nodes.yml
+* 03_setup_transport_node_profiles.yml
+* 02_setup_TEP_IP_Pools.yml
+* 01_setup_transport_zone.yml
 
 Validated against:
 * NSX-T 2.4 GA
+* NSX-T 2.5 GA
```
Comment (madhukark) [inline examples/setup_infra/README.md:26]: Add NSX-T 3.0 if its not too much trouble please

File: README.md [hunk 6]
```diff
@@ -65,42 +77,186 @@ This repository has different branches with each branch providing support for up
...
+### Install PyVmOmi
 ```
 pip install --upgrade pyvmomi pyvim requests ssl
 ```
-Download and Install Ovf tool 4.3 - [Ovftool](https://my.vmware.com/web/vmware/details?downloadGroup=OVFTOOL430&productId=742)
+### Download and Install Ovf tool 4.3 - [Ovftool](https://my.vmware.com/web/vmware/details?downloadGroup=OVFTOOL430&productId=742)
 (Note: Using ovftool version 4.0/4.1 causes OVA/OVF deployment failure with Error: cURL error: SSL connect error\nCompleted with errors\n)
 
-Download [ansible-for-nsxt](https://github.com/vmware/ansible-for-nsxt/archive/master.zip).
+### Download [ansible-for-nsxt](https://github.com/vmware/ansible-for-nsxt/archive/dev.zip).
```
Comment (ggverma) [inline README.md:117]: https://github.com/vmware/ansible-for-nsxt/archive/dev.zip
should be https://github.com/vmware/ansible-for-nsxt/archive/master.zip

**Pull request merged**